proprietary/design-tokens/studio/common/viewport (beta)/lg.json

```diff
@@ -0,0 +1,136 @@
+{
+  "figma": {
+    "rijkslint": {
+      "variant": {
+        "value": "default",
+        "type": "text",
+        "parent": "viewport (beta)/lg",
+        "description": ""
+      }
+    },
+    "screen-size": {
+      "value": "64rem",
+      "type": "dimension",
+      "parent": "viewport (beta)/lg",
+      "description": ""
+    }
+  },
+  "lux": {
+    "grid": {
+      "gutter": {
+        "value": "2rem",
+        "type": "dimension",
+        "parent": "viewport (beta)/lg",
+        "description": ""
+      },
+      "margin": {
+        "value": "2.5rem",
+        "type": "dimension",
+        "parent": "viewport (beta)/lg",
+        "description": ""
+      }
+    },
+    "size": {
+      "icon": {
+        "wrapper-height": {
+          "value": "1.75rem",
+          "type": "dimension",
+          "parent": "viewport (beta)/lg",
+          "description": ""
+        }
+      },
+      "target [oud]": {
+        "value": "3rem",
+        "type": "dimension",
+        "parent": "viewport (beta)/lg",
+        "description": ""
+      }
+    },
+    "font-size": {
+      "s": {
+        "value": "1rem",
+        "type": "dimension",
+        "parent": "viewport (beta)/lg",
+        "description": ""
+      },
+      "m": {
+        "value": "1.125rem",
+        "type": "dimension",
+        "parent": "viewport (beta)/lg",
+        "description": ""
+      },
+      "l": {
+        "value": "1.375rem",
+        "type": "dimension",
+        "parent": "viewport (beta)/lg",
+        "description": ""
+      },
+      "xl": {
+        "value": "1.625rem",
+        "type": "dimension",
+        "parent": "viewport (beta)/lg",
+        "description": ""
+      },
+      "2xl": {
+        "value": "2.5rem",
+        "type": "dimension",
+        "parent": "viewport (beta)/lg",
+        "description": ""
+      }
+    },
+    "line-height": {
+      "s": {
+        "value": "1.5rem",
+        "type": "dimension",
+        "parent": "viewport (beta)/lg",
+        "description": ""
+      },
+      "m": {
+        "value": "1.75rem",
+        "type": "dimension",
+        "parent": "viewport (beta)/lg",
+        "description": ""
+      },
+      "l": {
+        "value": "2rem",
+        "type": "dimension",
+        "parent": "viewport (beta)/lg",
+        "description": ""
+      },
+      "xl": {
+        "value": "2.25rem",
+        "type": "dimension",
+        "parent": "viewport (beta)/lg",
+        "description": ""
+      },
+      "2xl": {
+        "value": "3.5rem",
+        "type": "dimension",
+        "parent": "viewport (beta)/lg",
+        "description": ""
+      }
+    }
+  },
+  "viewport": {
```

Comment:
wordt nog opgepakt door de parser, resulterend in een `--viewport`-variabele

Reply:
Die zal ik toevoegen in #92. Deze tokens worden nergens gebruikt